docs/manuals/source/Appendix/OperatingSystems.rst

```diff
@@ -13,36 +13,36 @@ Additionally, the following table gives an overview on the available package ver
 .. csv-table::
    :header: "Operating Systems", "Version", "Client", "Director", "Storage"
 
-   :strong:`Linux`  :index:`\ <single: Platform; Linux>`
-   :index:`Arch Linux <single: Platform; Arch Linux>`, , `X <https://aur.archlinux.org/pkgbase/bareos/>`__,      `X <https://aur.archlinux.org/pkgbase/bareos/>`__, `X <https://aur.archlinux.org/pkgbase/bareos/>`__
-   CentOS, current, v12.4, v12.4, v12.4
-   Debian, current, v12.4, v12.4, v12.4
-   Fedora, current, v12.4, v12.4, v12.4
-   :index:`Gentoo <single: Platform; Gentoo>`, , `X <https://packages.gentoo.org/packages/app-backup/bareos>`__,     `X <https://packages.gentoo.org/packages/app-backup/bareos>`__, `X <https://packages.gentoo.org/packages/app-backup/bareos>`__
-   openSUSE, current, v12.4, v12.4, v12.4
-   (RH)EL, current, v12.4, v12.4, v12.4
-   SLES, current, v12.4, v12.4, v12.4
-   Ubuntu, current, v12.4, v12.4, v12.4
-   :ref:`Univention Corporate Linux <section-UniventionCorporateServer>`, current, v12.4, v12.4, v12.4
-   :ref:`Universal Linux Client <section-UniversalLinuxClient>`, openSSL 1.1, v21.0, v21.0, v21.0
-
-   :strong:`MS Windows`
-   :ref:`MS Windows <section-windows>` 32bit, 10/8/7, v12.4, v15.2, v15.2
-                                           , 2008/Vista/2003/XP, v12.4–v14.2
-   :ref:`MS Windows <section-windows>` 64bit, >= 7, v12.4, v15.2, v15.2
-                                            , 2008/Vista, v12.4–v14.2
-
-   :strong:`Mac OS`
-   :ref:`Mac OS X/Darwin <section-macosx>`, ,v14.2
-
-   :strong:`BSD`
-   :ref:`FreeBSD <section-FreeBSD>` :index:`\ <single: Platform; FreeBSD>`, >= 11, v19.2, v19.2, v19.2
-   OpenBSD, , X
-   NetBSD,  , X                                                                                                                                                            
-   :strong:`Unix`
-   :index:`AIX <single: Platform; AIX>`,         >= 4.3, com-13.2, \*, \*
-   :index:`HP-UX <single: Platform; HP-UX>`,           , com-13.2
-   :ref:`Solaris <section-Solaris>` (i386/Sparc) :index:`\ <single: Platform; Solaris>`, >= 11 , com-12.4, com-12.4, com-12.4
+   ":strong:`Linux`  :index:`\ <single: Platform; Linux>`"
+   ":index:`Arch Linux <single: Platform; Arch Linux>`", , `X <https://aur.archlinux.org/pkgbase/bareos/>`__, `X <https://aur.archlinux.org/pkgbase/bareos/>`__, `X <https://aur.archlinux.org/pkgbase/bareos/>`__
+   "CentOS", current, v12.4, v12.4, v12.4
+   "Debian", current, v12.4, v12.4, v12.4
+   "Fedora", current, v12.4, v12.4, v12.4
+   ":index:`Gentoo <single: Platform; Gentoo>`", , `X <https://packages.gentoo.org/packages/app-backup/bareos>`__, `X <https://packages.gentoo.org/packages/app-backup/bareos>`__, `X <https://packages.gentoo.org/packages/app-backup/bareos>`__
+   "openSUSE", current, v12.4, v12.4, v12.4
+   "(RH)EL (x86, arm64)", current, v12.4, v12.4, v12.4
```

Comment:
It look like this addition just push the table width out of the max page size and add a horizontal scrollbar (which I'm not fan at all, especially for 2 blanks chars...)
Maybe removing " and () we can keep the constraint, or removing x86, arm64 because we also don't specify those arch on plain packages...

Reply:
changed the table, so that it allows multi-lines.